### ISSUE TITLE
Exception in async_process_queue when dispatching 'scheduler_started' - v3.2.2

### ISSUE THREAD

**Issue by Masterz69** [nielsfaber/scheduler-component#147]
### Checklist
- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I am running the latest version.

### Expected behavior
See no messages in log on HA Core startup related to Scheduler integration.

### Actual behavior
On HA Core run :
``` txt
2021-11-16 20:26:25 ERROR (MainThread) [homeassistant.util.logging] Exception in async_process_queue when dispatching 'scheduler_started': ()
Traceback (most recent call last):
  File "/config/custom_components/scheduler/actions.py", line 248, in async_process_queue
    res = await self.async_execute_action(action, conditions, condition_type)
  File "/config/custom_components/scheduler/actions.py", line 263, in async_execute_action
    else any(validate_condition(self.hass, item) for item in conditions)
  File "/config/custom_components/scheduler/actions.py", line 263, in <genexpr>
    else any(validate_condition(self.hass, item) for item in conditions)
  File "/config/custom_components/scheduler/actions.py", line 138, in validate_condition
    result = actual < required
TypeError: '<' not supported between instances of 'str' and 'int'

```

First time noticed that message after Scheduler Integration was upgraded to v3.2.2 & HA Core to 2021.11.4.

Tried to change schedules - looks like all working fine.

**Comment by nielsfaber** [comment]
This is caused by a condition which is set up incorrect.
A number is being compared with a text.
Could you give some information about the schedule configuration?

**Comment by Masterz69** [comment]
Fortunately or not - from some time see no more this message in home-assistant.log.
Have only one schedule with condition on sensor.xxx_temperature below some value.
If appear again - will reopen issue.

P.S. scheduler entity does not contain condition, as far I see:
![image](https://user-images.githubusercontent.com/15225823/144720961-28329e57-6bb1-429a-b021-0b5c2f78a4aa.png)

Probably I've edited scheduler on December 03 and issues was fixed.
